CHANGELOG.md

```diff
@@ -27,6 +28,34 @@
  - [0.0.5](#005---20180602)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.0.0] - WIP
```

Comment:
```suggestion
## [2.0.0-alpha.1] - TBD
```
Align with the above. You'll also need to add a link ref at the bottom for the compare:
```
[2.0.0-alpha.1]: https://github.com/kong/kubernetes-ingress-controller/compare/1.2.0...2.0.0-alpha.1
```

Reply:
:+1: resolved with afcccdc932ea8ce88a033e8270f9bd2fbcebdf34